package.json

```diff
@@ -242,7 +242,7 @@
   "private": true,
   "dependencies": {
     "@babel/runtime": "^7.15.4",
-    "@department-of-veterans-affairs/component-library": "^13.8.0",
+    "@department-of-veterans-affairs/component-library": "^13.12.0",
```

Comment:
In the failing tests, it looks like several of them are not finding an id `error-message`. Since you are updating the version here, it's likely that this ID has changed in a component library release and you'll need to update the tests for it.

Reply:
@powellkerry I think you can either just merge `main` into this branch or make the same updates we made here to the tests: https://github.com/department-of-veterans-affairs/vets-website/pull/23347/files